### ISSUE TITLE
Delete/get info doesn't work anymore on M1 Monterey

### ISSUE THREAD

**Issue by albert-ying** [fman-community/fman#840]
The deleting and get info functions do not work even with permission

<img width="361" alt="image" src="https://user-images.githubusercontent.com/59846322/141181655-842cea2d-e7d9-44f8-a6a1-c984bf937e78.png">

**Comment by aberenyi** [comment]
Any update on this?

**Comment by albert-ying** [comment]
Maybe worth to look at this:

https://github.com/morgant/tools-osx#trash

> trash
> 
> trash allows trashing of files instead of tempting fate with rm. Correctly handles trashing files on other volumes, uses the same filename renaming scheme as Finder for duplicate file names, can list trash contents w/disk usage summary, and empty trash (including securely) w/confirmation. Does not require Finder to be running.

Hope this app is not abandoned.

**Comment by albert-ying** [comment]
@aberenyi I created a simple plugin to handle this. You can check it here https://github.com/albert-ying/trashselected

**Comment by cifarra** [comment]
Not working on Intel Monterey either.
The file is deleted and quickly comes back where it was.

**Comment by mherrmann** [comment]
Can your issues be resolved by following the instructions under "Deleting files on macOS" at the bottom of https://fman.io/docs/macos ?

**Comment by aberenyi** [comment]
@mherrmann Nope, however, I've noticed `Finder` is not present in the Automation section.

Is there a way to force the [security prompt](https://fman.io/static/public/img/catalina-fman-control-finder.png) again?

![image](https://user-images.githubusercontent.com/1775664/162406386-d602b975-f539-4edb-8270-c762c462a6b0.png)

**Comment by mherrmann** [comment]
@aberenyi I'm sorry, I don't know.

**Comment by pravic** [comment]
@aberenyi Same here.

According to https://discussions.apple.com/thread/253138286?sortBy=best, the only option is to uninstall `fman` and install it again, to trigger the privilege request again.

**Comment by niya3** [comment]
@aberenyi @mherrmann 

i've faced with the same issue and i think that i've found a workaround to call a security promt - it raised after i've pressed RMC on some file and selected "Get info" from context menu. After pressing OK **fman** finally appeared in "Autiomation" section.

jfyi: my setup - macOS Ventura 13.6 M1 and fman 1.7.3

**Comment by hokublack** [comment]
@aberenyi @mherrmann @niya3
MacBookAir M4 2025 -> faced the same problem, could not delete a file. Resolved by the recommendation from @niya3 > selected "Get info" on any file to call a security promt to be able to add fman in "Autiomation" section.
